test/MatrixFields/field_matrix_solvers.jl

```diff
@@ -298,7 +298,10 @@ end
     end
 
     @testset "approximate iterative solve with debugging" begin
-        Logging.with_logger(Logging.SimpleLogger(stderr, Logging.Debug)) do
+        logger =
+            using_cuda ? Logging.NullLogger() :
+            Logging.SimpleLogger(stderr, Logging.Debug)
+        Logging.with_logger(logger) do
```

Comment:
Do you need to set the logger here? I think `@test_logs` takes care of setting the logger internally.

Reply:
Yeah, this is how I got this to run